src/coreclr/.nuget/Directory.Build.props

```
@@ -89,6 +89,7 @@
   </ItemGroup>
   <ItemGroup Condition="$(SupportedPackageOSGroups.Contains(';osx;'))">
     <OfficialBuildRID Include="osx-x64" />
+    <OfficialBuildRID Include="osx-arm64" />
```

```suggestion
    <OfficialBuildRID Include="osx-arm64">
      <Platform>arm64</Platform>
    </OfficialBuildRID>
```

We add this metadata in linux (lines 78 and 81), not sure if it is really needed.